packages/manager/modules/pci/src/projects/project/sidebar/sidebar.less

```diff
@@ -0,0 +1,10 @@
+cloud-sidebar-project-list {
+  @import '~ovh-ui-kit/packages/oui-color/_variables';
+
+  .project-warning-icon {
```

Comment:
Can't we use current icons ? Especially "legacy" version of `ui-kit` was recently updated with a light theme

Reply:
We are using the existing icon... This just modifies the style a bit to suit its location on the sidebar